Tests/SwiftGenKitTests/TestsHelper.swift

```diff
@@ -214,7 +245,7 @@ func XCTDiffContexts(
   if ProcessInfo().environment["GENERATE_CONTEXTS"] == "YES" {
     let target = Path(#file).parent().parent() + "Fixtures/StencilContexts" + directory.rawValue + fileName
     do {
-      try YAML.write(object: result, to: target)
+      try YAML.write(object: result.convertedToContext(), to: target)
```

Comment:
@djbe I've pushed this fix on this PR because I discovered the crash while working on this, but do you think it's worth cherry-picking in develop via a separate PR?

---

FYI, basically before this fix, the "Generate Contexts" scheme crashed because YAMS was failing to convert the `StencilContextLazyDocument` object into a yaml `Node`; I tried to make `StencilContextLazyDocument` conform to `NodeRepresentable` but that let to more complexity (having to deal with `Node.Mappings` and friends) hence this solution instead.